grpc-proxy/proxy-server/oidc.go

```diff
@@ -0,0 +1,62 @@
+package main
```

Comment:
Ideally we would have tests confirm this works and will work into the future. But I understand if it's not something you want to do right now.